### ISSUE TITLE
No prompt to connect to Infura

### ISSUE THREAD

**Issue by pcko1** [microsoft/vscode-azure-blockchain-ethereum#71]
### Problem

I get no interactive prompt to type in my credentials to connect to Infura when clicking `Sign in to Infura account`.

#### Local

- VS Code `1.55.0-insider`
- Local machine: macOS Big Sur `11.2.1` (M1 chip)

#### Remote

- Remote machine (through `Remote-SSH`): Ubuntu `20.04`
- Blockchain Development Kit for Ethereum `1.6.0`

### Steps to replicate

After connecting to remote machine via Remote-SSH I press:

<img width="599" alt="Screenshot 2021-03-26 at 19 36 50" src="https://user-images.githubusercontent.com/30870180/112678060-c81db600-8e6a-11eb-918b-7427df89a16c.png">

I get no interactive prompt to login whatsoever and on the bottom of the screen I see:

<img width="338" alt="Screenshot 2021-03-26 at 19 37 20" src="https://user-images.githubusercontent.com/30870180/112678184-ea173880-8e6a-11eb-9dfb-5bb0e07ce792.png">

This keeps pending until it times out. If I click on it before timing out, I get:

<img width="471" alt="Screenshot 2021-03-26 at 19 37 34" src="https://user-images.githubusercontent.com/30870180/112678230-f69b9100-8e6a-11eb-9a5f-be5065c08907.png">

**Comment by caleteeter** [comment]
Thank you @pcko1 for this.  I just validated the login with Infura, albeit on a Windows based machine with success.  The proper flow would be:

1.  Initiate the login to Infura, as you did above.
2. The users default browser is then opened navigating to the infura login.
3. User logs in with their credentials and on success is then told to close the browser and come back to the IDE.
4. User sees the login in the status bar and can use the projects / integration to Infura.

The problem may be with Mac platform specifically, I will need to do some more testing there to confirm.

A short demo of this is below:
![interact](https://user-images.githubusercontent.com/8452367/112692719-c5649600-8e55-11eb-99b8-fbd1b758dde9.gif)

I did notice that a new captcha has been added to the login with Infura that does seem to be hidden, which means you need to login to Infura out of band and then login via the extension.  I will investigate this on another thread.

**Comment by pcko1** [comment]
I think the problem is the remote Ubuntu server that I am SSHing to. I found out that the supported OS for `Blockchain Development Kit for Ethereum` is either Windows or macOS:

<img width="338" alt="Screenshot 2021-03-27 at 11 18 48" src="https://user-images.githubusercontent.com/30870180/112717653-6868ee80-8eee-11eb-85cd-d6574c63e35e.png">

sad :( :(

I have just installed it locally on my Mac (without firstly connecting to my remote Ubuntu machine via Remote-SSH extension) and the behavior is nominal (similar to the one you showcased above). Would be awesome to support Ubuntu in the future :)

**Comment by caleteeter** [comment]
Ah, ok, I think we are limited there because we are popping the default browser from the VS Code extension api.  I will look into this, and see if we have any alternatives.  Worst case we could possibly give you link and ask you to go login with a code (device code flow).

**Comment by greglnye** [comment]
I have the same issue but am using Ubuntu 20.04 via WSL 2 on Windows 10.

**Comment by alexrainman** [comment]
@caleteeter is this extension being abandonware?

**Comment by caleteeter** [comment]
Not at all, we are working a new version of this.  Apologies on the lag on response here, not great.

**Comment by alexrainman** [comment]
@caleteeter thanks for a quick response. The more important functionality of this extension, debugging Smart Contracts, is broken, so when that release will be out?
